js/handlers.js

```diff
@@ -248,6 +248,11 @@ weechat.factory('handlers', ['$rootScope', '$log', 'models', 'plugins', 'notific
             // Update indentation status
             old.type = localvars.type;
             old.indent = (['channel', 'private'].indexOf(localvars.type) >= 0);
+            // Update serverSortKey and related variables
+            old.plugin = localvars.plugin;
```

Comment:
I don't think `plugin` can ever change and it's also known when the buffer is first created (unlike `server`) so I'm not sure this is necessary